### PR TITLE
In Car Audio

### PR DESCRIPTION
**Pull request by askirk**

### CODE REVIEW

File: docs/Getting In-Car Audio/index.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+```
+
+!!! note
+This audio data is only the current audio data, so the developer must be in charge of managing previously retrieved audio data.
+!!!
+
+
+### Ending Audio Capture
+Perform Audio Pass Thru is a request that works in a different way when compared to other RPCs. For most RPCs, and request is followed by an immediate response, with whether that RPC was successful or not. This RPC, however, will only send out the response when the Perform Audio Pass Thru is ended.
+
```
Comment (joeygrover) [inline docs/Getting In-Car Audio/index.md]: `For most RPCs, and request` should be  `For most RPCs a request`

Comment (joeygrover) [inline docs/Getting In-Car Audio/index.md]: `immediate response, with whether that RPC was successful or not` should be `immediate response that informs the developer whether or not that RPC was successful`

File: docs/Getting In-Car Audio/index.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+Audio Capture can be ended in 4 ways:
+
+1. Audio Pass Thru has timed out.
+
+    If the audio passthrough has proceeded longer than the requested timeout duration, Core will end this request and send a `PerformAudioPassThruResponse` with a `Result` of `SUCCESS`. You should expect to handle this audio passthrough as though it was successful.
+
+2. Audio Pass Thru was closed due to user pressing "Cancel".
+
+    If the audio passthrough was displayed, and the user pressed the "Cancel" button, you will receive a `PerformAudioPassThruResponse` with a `Result` of `ABORTED`. You should expect to ignore this audio pass through.
```
Comment (joeygrover) [inline docs/Getting In-Car Audio/index.md]: The reference to AudioPassThru should remain consistent. I would suggest using it as a single word, `AudioPassThru`. This goes for the items under this one as well.

File: docs/Getting In-Car Audio/index.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+EndAudioPassThru endAPT = new EndAudioPassThru();
+endAPT.setCorrelationID(CorrelationIdGenerator.generateId());
+proxy.sendRPCRequest(endAPT);
+```
+
+You will receive a `EndAudioPassThruResponse` and `PerformAudioPassThruResponse` with a `Result` of `SUCCESS`, and should expect to handle this audio passthrough as though it was successful.
+
+### Handling the Response
+To process the response that we received from an ended audio capture, we monitor the `PerformAudioPassThruResponse` through the callback in the Sdl Service.
+
```
Comment (joeygrover) [inline docs/Getting In-Car Audio/index.md]: Should we add something that at this time the developer should have received all the audio data so they can begin processing?

File: docs/Getting In-Car Audio/index.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+proxy.sendRPCRequest(performAPT);
+```
+
+#### Ford HMI
+![Ford Audio Pass Thru](assets/Ford_AudioPassThruPrompt.png)
+
+In order to know the currently supported audio capture capabilities of the connected head unit, please refer to the `SdlProxyALM` object created when your SDL Service starts. It contains a method `getAudioPassThruCapabilities` that returns a list of `AudioPassThruCapabilities` that the head unit supports.
+
+!!! note
+Currently, Ford's SYNC 3 vehicles only support Sampling Rates of 16 khz and Bit Rates of 16 bit.
```
Comment (joeygrover) [inline docs/Getting In-Car Audio/index.md]: `Sampling Rates of.....Bit Rates` should be `a sampling rate of..... a bit rate`

File: docs/Getting In-Car Audio/index.md (hunk 3 above)
Comment (askirk) [inline docs/Getting In-Car Audio/index.md]: I was thinking- 

"If the response has a successful `Result`, all of the audio data for the passthrough has been received and is ready for processing."

**Pull request merged**